.github/workflows/release-beta.yml

```diff
@@ -0,0 +1,67 @@
+name: Release (Beta)
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: semantic-release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    outputs:
+      new-release-published: ${{ steps.semantic-release.outputs.new_release_published }}
+      new-release-version: ${{ steps.semantic-release.outputs.new_release_version }}
+    steps:
+      - uses: actions/checkout@v3
+      - id: semantic-release
+        uses: cycjimmy/semantic-release-action@v3
```

Comment:
v3 action now [pins to 19.0.5](https://github.com/cycjimmy/semantic-release-action#semantic_version) by default